xclim/sdba/properties.py

```diff
@@ -307,21 +337,21 @@ def _spell_stats(ds, *, dim, method, thresh, op, freq, stat):
         freq=group.freq,
         stat=stat,
     ).out
-    out.attrs.update(attrs)
-    out.attrs[
-        "long_name"
-    ] = f"{stat} of spell length when input variable {op} {method} {thresh}"
-    out.name = "spell_length_distribution"
-    out.attrs["units"] = "day"
+    out.attrs["units"] = to_agg_units(out, da, op="count")
```

Comment:
```suggestion
    out.attrs["units"] = to_agg_units(out, da, op="count").units
```
to_agg_units outputs a dataset... I think this is what you wanted to do?

Reply:
Indeed, woups.